### DISCUSSION TITLE
使用 Build App with Pake Cli 构建的

### DISCUSSION THREAD

**Discussion by buildnewapp** [tw93/Pake#431] [General]
直接Fork，然后运行Action
提示：
Annotations
3 errors
build (windows)
Error 403: Resource not accessible by integration
build (linux)
Error 403: Resource not accessible by integration
build (macos)

build (windows)Error 403: Resource not accessible by integration
--
build (linux)Error 403: Resource not accessible by integration
build (macos) Error 403: Resource not accessible by integration

Run ncipollo/release-action@v[1]
Error: Error 403: Resource not accessible by integration
Run ncipollo/release-action@v1
Error: Error 403: Resource not accessible by integration
Run ncipollo/release-action@v1
Error: Error 403: Resource not accessible by integration

请问大家正常运行吗？

**Comment by Tlntin** [comment]
参考这个。https://github.com/tw93/Pake/issues/440#event-9270065937
